aws/resource_aws_db_instance.go

```diff
@@ -674,7 +674,7 @@ func resourceAwsDbInstanceCreate(d *schema.ResourceData, meta interface{}) error
 			return nil
 		})
 		if err != nil {
-			return fmt.Errorf("Error creating DB Instance: %s", err)
+			return fmt.Errorf("Error creating DB Instance: provided name: %v, %s", *opts.DBName, err)
```

Comment:
This will likely cause a panic as usually the response from the SDK when there is an error is `(nil, error)`.